apps/aehttp/test/aehttp_integration_SUITE.erl

```diff
@@ -298,8 +298,7 @@ groups() ->
        {group, swagger_validation},
        {group, wrong_http_method_endpoints},
        {group, naming},
-       {group, channel_websocket},
-       {group, channel_websocket_legacy}
```

Comment:
Regardless of when this PR is merged, it would be helpful to have an integration test checking what the client sees when attempting to open state channel using legacy protocol. This may require hand-rolling request - if test helpers for legacy protocol are being removed - and if so it may be too expensive: if so, please feel free to just add github comment in this PR (or comment in Pivotal Tracker) with result of manual testing.